corsheaders/models.py

```diff
@@ -3,3 +3,6 @@
 
 class CorsModel(models.Model):
     cors = models.CharField(max_length=255)
+    
+    def __unicode__(self)
```

Comment:
Please swap this for `__str__` and use `@python2_unicode_compatible` as per https://docs.djangoproject.com/en/1.10/topics/python3/#str-and-unicode-methods. Also add a line in `HISTORY.rst` 😄